DragaliaAPI/Controllers/Dragalia/DungeonRecordController.cs

```diff
@@ -49,7 +65,23 @@ public async Task<DragaliaResult> Record(DungeonRecordRecordRequest request)
         userData.Exp += 1;
         userData.ManaPoint += 1000;
         userData.Coin += 1000;
-        userData.Crystal += 25;
+
+        bool[] clearedMissions = new bool[5]
+        {
+            isFirstClear,
+            oldMissionClear1 != newQuestData.IsMissionClear1 && newQuestData.IsMissionClear1,
```

Comment:
would it be clearer to write 
```suggestion
            newQuestData.IsMissionClear1 && !oldMissionClear1,
```

Reply:
Oh yeah will change those tyty